cli/build.go

```diff
@@ -145,8 +144,7 @@ func (b *buildCmd) run(cmd *cobra.Command, args []string) error {
 	// TODO: create secrets
 	secrets := []*graph.Secret{}
 
-	p := graph.NewPipeline(steps, push, secrets)
-	dag, err := graph.NewDagFromPipeline(p)
+	p, err := graph.NewPipeline(steps, push, secrets, b.opts.Registry, b.registryUser, b.registryPw)
```

Comment:
>p [](start = 1, length = 1)

rename to pipeline? I often need to look up to find what p means if I see p is passed as parameter

Reply:
renamed all instances

Reply:
Er... looks like my `push` didn't go through because GH is having status up issues... and I merged already, I'll combine it with the next PR.